Prevent instructions that are unsupported by older CPUs from being used

--- a/src/d/llvm/backend.d
+++ b/src/d/llvm/backend.d
@@ -44,7 +44,7 @@ public:
 			LLVMGetFirstTarget(),
 			LLVMGetDefaultTargetTriple(),
 			LLVMGetHostCPUName(),
-			LLVMGetHostCPUFeatures(),
+			"",
 			LLVMCodeGenOptLevel.Default,
 			Reloc,
 			LLVMCodeModel.Default,
